### DISCUSSION TITLE
Synchronization/Timing in K8s+Linux platforms

### DISCUSSION THREAD

**Discussion by taylor** [lfn-cnti/bestpractices#174] [General]
What are the requirements for timing? What are the qualifications for real-time?

What is available in K8s? What is available on default linux host systems? 

Ref: 
- https://github.com/cncf/cnf-wg/pull/168
  - UC-0001: Synchronization/Timing in Telco CNF world
  - UC-0002: Support of Multi-Communication Interface in Telco CNF world

**Comment by taylor** [comment]
Ubuntu on real-time kernels for NFV (2018) - https://ubuntu.com/blog/low-latency-real-time-kernels-telco-nfv

**Comment by taylor** [comment]
K3os did have real-time kernel support in the past.  Not sure now. https://github.com/rancher/k3os

**Comment by taylor** [comment]
[OpenNESS](https://www.openness.org/) (Open Network Edge Services Software) has real-time and low latency focus:
- https://www.openness.org/docs/doc/building-blocks/enhanced-platform-awareness/openness-node-feature-discovery
- [OpenNESS: Enabling High Performance Edge for Telco & Enterprise](https://www.cncf.io/wp-content/uploads/2020/11/UPDATED-intel.kubernetes.in_.the_.context.of_.on-premises.edge_.and_.network.edge_.computing_openness-1.pdf) (PDF - 11/2020)

**Comment by taylor** [comment]
OpenShift Operator - Performance Addon Operator for low latency nodes
- https://docs.openshift.com/container-platform/4.6/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html

**Reply by tliron** [reply]
Also see ACM: https://www.redhat.com/en/technologies/management/advanced-cluster-management

**Comment by taylor** [comment]
@fkautz: what are the ORAN people doing?

cc: @tliron
